lib/router.js

```diff
@@ -282,7 +282,7 @@ module.exports = class Router extends express.Router {
           res.end('<script>window.close();</script>');
         })
         .catch((err) => {
-          res.status(SYSTEM_ERROR_STATUS).send(err);
+          res.end(`<script>window.opener.postMessage('${JSON.stringify(err)}', '*'); window.close();</script>`);
```

Comment:
Mozilla tells that providing "*" for domain parameter is a very bad practice but I did not know what I could provide in place. A complete URI like Paht.join(dirname, ...) ?

Reply:
you can provide the domain we are on